### DISCUSSION TITLE
Website does not work for certain ISPs

### DISCUSSION THREAD

**Discussion by Uche01** [vercel/community#6009] [Help]
### Summary

I have a very weird issue that has kept my site inaccessible  for over 6 hours, The website does not load for certain internet service provider eg MTN & Airtel but works when I use Glo.

The issue is not from my DNS as I've confirmed that with Namecheap, also I have done a redeploy but issue still persist. I have also tried to upgrade my plan but keep getting weird error "Error: String expected but found null" in the billing form. Really can't decipher how to fixed this issue.

PS: I'm based in Nigeria

### Steps to Reproduce

No specific step, my website domain returns "Site can't be reached error" when accessed with certain ISPs

**Comment by overtoo** [comment]
My users in Nigeria cannot access my Vercel-hosted website. Traffic from Nigeria has dropped 80% since March 3rd.

**Comment by amyegan** [comment]
@Uche01 @overtoo Thanks for reaching out. Are you still experiencing this problem?

**Reply by Uche01** [reply]
Yes @amyegan and this has been frustrating. For 3 days I can't access my website. Infact we are already exploring other options. This is really sad.

**Reply by Uche01** [reply]
I also tried to upgrade to pro plan but kept getting an unknown error: "Expected String but found null" in the checkout modal

**Reply by amyegan** [reply]
Your site is not the only one affected. These sites are being blocked by some ISPs. We reached out to them for help with this, but we don't have much control over when the IP addresses will be unblocked. 

Thank you for your patience while we work with them to get this solved. 🙇 

Upgrading to Pro would not solve this issue, but the team also looked into that error and released a solution. Please let me know if you run into any more trouble with the Pro upgrade process.

**Comment by WorkMender** [comment]
We are also facing the same issue. However, the account was made in Australia whereas our team is unable to access the website in Nigeria. Our domain is workmender.com.
